java/java.source.base/src/org/netbeans/modules/java/source/pretty/ImportAnalysis2.java

```diff
@@ -390,7 +390,7 @@ private boolean checkPackagesForStarImport(String pkgName, CodeStyle cs) {
     }
 
     private PackageElement getPackageOf(Element el) {
-        while (el.getKind() != ElementKind.PACKAGE) el = el.getEnclosingElement();
+        while ((el != null) && (el.getKind() != ElementKind.PACKAGE)) el = el.getEnclosingElement();
```

Comment:
More style than anything else but shouldn't a new variable and enclosing bracket makes the code more readable ?

Reply:
Not really. This is just a "one liner" private method